### PR TITLE
Kk wdac edits task 33881267

### PR DESCRIPTION
**Pull request by v-kikl**
@jgeurten, is this what you had in mind?

### CODE REVIEW

**Review by jgeurten — APPROVED** [review]
This is perfect. Thank you, Kim

File: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md [hunk 1]
```diff
@@ -27,6 +27,13 @@ ms.technology: mde
 
 Windows Defender Application Control (WDAC) can control what runs on Windows 10 by setting policies that specify whether a driver or application is trusted. A policy includes *policy rules* that control options such as audit mode, and *file rules* (or *file rule levels*) that specify how applications are identified and trusted.
 
+## Getting started with commdlets
+Some of the [SKUs](feature-availability.md) that support our PowerShell commandlets [(ConfigCI Module | Microsoft Docs)](powershell/module/configci) support but do not have the module installed on the box.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md:31]:
```suggestion
Some of the [SKUs](feature-availability.md) that support our PowerShell commandlets [(ConfigCI Module)](powershell/module/configci/?view=windowsserver2019-ps) support but do not have the module installed on the box. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-kikl, please commit my suggested change if you agree with it. Many thanks.

**Comment by v-kikl** [comment]
Updated the link and fixed merge conflict.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**